src/psyclone/psyir/transformations/omp_taskwait_trans.py

```diff
@@ -164,19 +164,17 @@ def validate(self, node, options=None):
                 # safely guarantee our forward dependency so throw an
                 # error
                 if not valid:
-                    fwriter = FortranWriter()
+                    fwr = FortranWriter()
                     # pylint: disable=cell-var-from-loop
-                    raise TransformationError(
-                                LazyString(lambda: "Couldn't satisfy the "
-                                                   "dependencies due to "
-                                                   "taskloop dependencies "
-                                                   "across barrierless OMP "
-                                                   "serial regions. Dependency"
-                                                   " is from\n{0}\nto\n{1}"
-                                           .format(
-                                               fwriter(taskloop).rstrip("\n"),
-                                               fwriter(forward_dep)
-                                                   .rstrip("\n"))))
+                    raise TransformationError(LazyString(
```

Comment:
Please add a comment here explaining why you have `chr(10)`. Could you have `'\n'` instead?

Reply:
Great, thanks.